build_scripts/build_mgard_serial.sh

```diff
@@ -9,6 +9,8 @@
 set -e
 set -x
 
+module load cmake
+
```

Comment:
The Summit build script `build_mgard_cuda_summit.sh` already builds both the CUDA and Serial backend in MGARD-X. This `build_mgard_serial.sh` is solely for building just the Serial backend on an arbitrary machine (e.g., any machine with only CPUs), so I'm not sure it is a good idea to have `module load` here.

Reply:
Oh, I didn't notice that `build_mgard_cuda_summit.sh` builds the serial backend. I will just delete this whole pull request, then. Thank you for reviewing, @JieyangChen7!